src/gt4py/next/program_processors/codegens/gtfn/gtfn_module.py

```diff
@@ -39,6 +40,11 @@ def get_param_description(name: str, obj: Any) -> interface.Parameter:
     return interface.Parameter(name, type_translation.from_value(obj))
 
 
+class HardwareAccelerator(enum.IntEnum):
+    CPU = enum.auto()
+    GPU = enum.auto()
```

Comment:
In the meantime there is a `DeviceKind` enum in `gt4py._core.definitions`. Please check if it makes sense to use it here.